doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be open source for users.
+6. Should be able to handle wrong input formats.
```

Comment:
For number 6, you may be more specific on how to `handle` it. e.g. `without restarting/closing the app`